Gopkg.lock

```diff
@@ -526,7 +526,7 @@
   version = "v1.3.2"
 
 [[projects]]
-  digest = "1:fd58696ccbfea94bb5773204f42c11416197587a1d22da198c251bcee859e572"
+  digest = "1:5e70aa639271a54e9ed87875b9f7e495255df54b1a64c7d4a338e33b44ee33b7"
```

Comment:
You also need to stash this change since you don't change Gopkg.toml anymore

Reply:
done